Dungeon.py

```diff
@@ -1,6 +1,6 @@
 class Dungeon(object):
 
-    def __init__(self, world, name, hint, font_color, boss_key, small_keys, dungeon_items):
+    def __init__(self, world, name, hint, font_color, boss_key=None, small_keys=None, dungeon_items=None):
         def to_array(obj):
```

Comment:
can now be removed

Reply:
It's still used in `copy`.

Reply:
uh... what? How can it be? `to_array` is a local to `__init__`, and not used anymore?

Reply:
Resolved.

Reply:
The review showing the lines above confused the hell out of me as to what you were referring to.